recipes/mayavi/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "4.5.0" %}
+
+package:
+  name: mayavi
+  version: {{ version }}
+
+source:
+  fn: mayavi-{{ version }}.tar.gz
+  url: https://github.com/enthought/mayavi/archive/{{ version }}.tar.gz
+  sha256: 36f688b3ea542e9f8cc0d7faa25e1425723cd00acc8aa640169029f33679ab85
+
+build:
+  entry_points:
+    - mayavi2 = mayavi.scripts.mayavi2:main
+    - tvtk_doc = tvtk.tools.tvtk_doc:main
+  osx_is_app: True
+
+requirements:
+  build:
+    - python
+    - numpy
+    - setuptools
+    - vtk
+    - traitsui
+    - apptools
+  run:
+    - python
+    - python.app             # [osx]
+    - numpy
+    - vtk
```

Comment:
Ah, I didn't kwon that....

http://www.vtk.org/Wiki/VTK/Python_Wrapping_FAQ#Python_Versions:

    Versions of VTK prior to 7.0 will only work with Python 2.